frontend/src/pages/PassingTest/PassingTestContainer.tsx

```diff
@@ -22,21 +22,21 @@ const PassingTestContainer: React.FC = () => {
   const courseTest = courseTestData?.length ? courseTestData[0].test : undefined;
   const maxStage = courseTest ? courseTest.questions.length : MAX_STAGE_INITIAL;
 
-  const [isOpenedTimeIsOverDialog, seTIsOpenedTimeIsOverDialog] = timeIsOverDialogToggle(false);
+  const [isOpenedTestTimeotDialog, setIsOpenedTestTimeotDialog] = useToggle(false);
```

Comment:
Maybe this one will be more readable?
`const [isTestTimeoutDialogOpen, setTestTimeoutDialogOpen] = useToggle(false);`